lib/tasks/batch_geocode.rake

```diff
@@ -0,0 +1,12 @@
+desc "Batch geocode models"
+namespace :geocoding do
+  task :batch_update, [:class] => :environment do |_task, args|
+    klass = args[:class]
+    model = klass.classify.safe_constantize
+
+    puts "Starting batch geocoding for #{klass}"
+
+    ids = model.where(latitude: nil).or(model.where(longitude: nil)).pluck(:id)
+    ids.each { |id| GeocodeJob.perform_later(klass, id) }
```

Comment:
This is just processing _all_ the _model_ records that need to be geocoded ... does the geocoder do it's own batching to limit how much we hit Google?

tbh, I'm a little bit uncomfortable with having this outside of `mcb` ... one of the goals of `mcb` was to contain all these kinds of tasks. It won't be hard to create a new command for this, and even if we don't implement the `-E` option at least it'd still be in `mcb`.

Reply:
This is a one-off though so wouldnt a rake task or migration would be more appropriate?